clarify how sentry-cli associates commits (#5188)
getsentry/sentry-docs: src/docs/product/releases/suspect-commits/index.mdx

sentry-cli releases set-commits --auto $VERSION
```

> If you don't have a repo-based integration associated with your Sentry organization, then the `--auto` flag will automatically use the git tree of your local repo, and associate commits between the previous release's head commit and the current head commit with the release. The `--auto` flag gets the identity of the remote <what?> locally, but pulls the identity of the current `head` and the contents of the commit from the remote. If this is the first release, Sentry will use the latest 10 - 20 commits, depending on the integration. This behavior is configurable with the `--initial-depth` flag.

Author: ```suggestion
If you don't have a repo-based integration associated with your Sentry organization, then the `--auto` flag will automatically use the git tree of your local repo, and associate commits between the previous release's head commit and the current head commit with the release. If this is the first release, Sentry will use the latest 10 - 20 commits, depending on the integration. This behavior is configurable with the `--initial-depth` flag. 

Alternatively, you can use the `--local` flag to enable this behavior by default.

```
